flashinfer_trace/tests/references/test_gemm_qwen3_4b.py

```
@@ -0,0 +1,147 @@
+NUM_KV_HEADS = 8
+INTERMEDIATE_SIZE = 9728
+
+
+def load_definition(name: str) -> Definition:
+    """Load a definition by name from definitions directory."""
+    for op_dir in DEFINITIONS_DIR.iterdir():
+        if op_dir.is_dir():
+            def_file = op_dir / f"{name}.json"
+            if def_file.exists():
+                return load_json_file(Definition, def_file)
+    raise FileNotFoundError(f"Definition {name} not found in {DEFINITIONS_DIR}")
```

Ruff (0.15.5)

[warning] Avoid specifying long messages outside the exception class

(TRY003)